app/lib/actions/candlepin/import_pool_handler.rb

```diff
@@ -26,6 +26,8 @@ def handle(message)
           message_handler.import_pool
         when /compliance\.created/
           reindex_consumer(message_handler)
+        when /system_purpose_compliance\.created/
+          reindex_consumer(message_handler)
```

Comment:
I thought I was going to be clever and tell you to update the installer but you already did, https://github.com/theforeman/puppet-katello/pull/258

:1st_place_medal: